### PR TITLE
Fix default momentum value of BatchNorm2d in MaxViT from 0.99 to 0.01

### PR DESCRIPTION
**Pull request by NicolasHug**
Closes https://github.com/pytorch/vision/issues/8250

### CODE REVIEW

**Review by vfdev-5 — APPROVED** [review]
lgtm
